doc/api/util.md

```diff
@@ -356,7 +357,9 @@ stream.on('data', (data) => {
 stream.write('With ES6');
 ```
 
-## util.inspect(object[, options])
+
+##  util.inspect(object[, options])
+## util.inspect(object[, showHidden][, depth][, colors])
```

Comment:
So should not this be `## util.inspect(object[, showHidden[, depth[, colors]]])`?

Reply:
I am doing all these changes and squashing the commits into one